### PR TITLE
Create fabric8-ui-admin-console.yaml

### PR DESCRIPTION
**Pull request by rohitkrai03**
Depends on quay repo creation as tracked in https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2538

### CODE REVIEW

**Comment by pbergene** [comment]
[test]

**Comment by rohitkrai03** [comment]
@pbergene The build is failing with these errors - 
```
['ERROR_AUTH', u'quay.io/openshiftio/rhel-fabric8-ui-admin-console:e1df25', 'time="2019-01-30T11:32:13Z" level=fatal msg="Error reading manifest e1df25 in quay.io/openshiftio/rhel-fabric8-ui-admin-console: manifest unknown: manifest unknown" \n']
['ERROR_NOAUTH', u'quay.io/openshiftio/rhel-fabric8-ui-admin-console:e1df25', 'time="2019-01-30T11:32:14Z" level=fatal msg="Error reading manifest e1df25 in quay.io/openshiftio/rhel-fabric8-ui-admin-console: unauthorized: access to the requested resource is not authorized" \n']
```

Is there something that i need to do on my side?

**Comment by pbergene** [comment]
[test]

**Comment by rohitkrai03** [comment]
[test]

**Comment by jmelis** [comment]
[test]

File: dsaas-services/fabric8-ui-admin-console.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+services:
+- hash: e1df2527134e449cbe2afb55231101a8c8538488
+  name: fabric8-ui-admin-console
+  path: /openshift/fabric8-admin-console.app.yaml
+  url: https://github.com/fabric8-ui/fabric8-admin-console/
+  hash_length: 6
```
Comment (jmelis) [inline dsaas-services/fabric8-ui-admin-console.yaml]: Please set `hash_length: 7`, that's the reason why this is failing.

Comment (rohitkrai03) [reply dsaas-services/fabric8-ui-admin-console.yaml]: Updated

**Comment by rohitkrai03** [comment]
@jmelis Can we merge this?

**Comment by rohitkrai03** [comment]
@jmelis Can you merge this?

**Comment by riuvshin** [comment]
[test]

**Comment by riuvshin** [comment]
[e2e-test-beta]
[e2e-test-released]

**Comment by riuvshin** [comment]
[e2e-test-beta]
[e2e-test-released]

**Comment by rohitkrai03** [comment]
[test]

**Comment by rohitkrai03** [comment]
[e2e-test-beta]
[e2e-test-released]

**Comment by riuvshin** [comment]
[e2e-test-beta]
[e2e-test-released]

**Pull request merged**